src/framework/parsers/ply.js

```diff
@@ -404,10 +404,29 @@ const readPly = async (reader, propertyFilter = null) => {
     // allocate element storage
     elements.forEach((e) => {
         e.properties.forEach((p) => {
+            p.read = (streamBuf) => {
```

Comment:
Please store the read functions in a separate array (which can be referenced by property index) instead of assigning it temporarily to the property instance.